lib/logstash/environment.rb

```diff
@@ -57,6 +57,22 @@ def test?
     # of some plugins dependedant gems.
     def set_gem_paths!
       ENV["GEM_HOME"] = ENV["GEM_PATH"] = logstash_gem_home
+      ENV["BUNDLE_GEMFILE"] = GEMFILE_PATH
```

Comment:
As I could check at the beginning of the file, GEMFILE_PATH is a pointer to the file named tools/Gemfile. I get this fix the workflow for the general use case, but I don't think this would fix whenever we use for example the file tools/Gemfile.plugins.test in order to run the tests.

Can we make this ENV variable able to be updated from outside, with a fallback to the default one. Like this for the other Gemfile we can also have the very same workflow.